### ISSUE TITLE
Resource names of dependent charts are no longer prefixed with the release name since common 3.0.x

### ISSUE THREAD

**Issue by roelofjansen** [k8s-at-home/library-charts#59]
# Details

**Helm chart name and version:**

common 3.0.1
bookstack 2.0.0

**What steps did you take and what happened:**

When deploying the new version of bookstack with enabled mariadb the release name of the mariadb statefulset is no longer "bookstack-mariadb" but simply "mariadb".

```bash
helm template --name-template bookstack .
```

```yaml
# Source: bookstack/charts/mariadb/templates/primary/statefulset.yaml
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: mariadb
```

**Relevant Helm values:**
```yaml
# Enabled mariadb
# ... for more options see https://github.com/bitnami/charts/tree/master/bitnami/mariadb
mariadb:
  enabled: true
```

**What did you expect to happen:**

```yaml
# Source: bookstack/charts/mariadb/templates/primary/statefulset.yaml
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: bookstack-mariadb
```

**Anything else you would like to add:**

The MariaDB chart also uses a "common.names.fullname". The new common-3.x define seems to conflict with it somehow.

**Additional Information:**

Helm v3.5.4 used.

**Comment by bjw-s** [comment]
Yikes, that's not good... will investigate and see what is happening. Thanks for reporting this!

**Comment by bjw-s** [comment]
I have found the underlying issue. In our v3 refactor we made some changes to the `common.names.fullname` template. Unfortunately we introduced a bug there where the `$name` variable wasn't actually being upated. This will be fixed with #60 .
